packages/react/src/components/Breadcrumbs/BreadcrumbsList.tsx

```diff
@@ -0,0 +1,42 @@
+import { inner, useMergeRefs } from '@floating-ui/react';
+import { Slot } from '@radix-ui/react-slot';
+import cl from 'clsx/lite';
+import { type HTMLAttributes, forwardRef, useEffect, useRef } from 'react';
+
+export type BreadcrumbsListProps = {
+  /**
+   * Change the default rendered element for the one passed as a child, merging their props and behavior.
+   * @default false
+   */
+  asChild?: boolean;
+} & Omit<HTMLAttributes<HTMLOListElement>, 'size'>;
+
+export const BreadcrumbsList = forwardRef<
+  HTMLOListElement,
+  BreadcrumbsListProps
+>(({ asChild, className, ...rest }, ref) => {
+  const Component = asChild ? Slot : 'ol';
+  const innerRef = useRef<HTMLOListElement>(null);
+  const mergedRefs = useMergeRefs([innerRef, ref]);
+
+  // Set aria-current on last link
+  useEffect(() => {
+    const links = innerRef.current?.querySelectorAll('a') || [];
+    const lastIndex = links?.length - 1;
+
+    links?.forEach((link, index) => {
+      if (index === lastIndex) link.setAttribute('aria-current', 'page');
+      else link.removeAttribute('aria-current');
```

Comment:
Is this `else` not redundant?. Suggest also giving it some space so its easier to read :) 

```suggestion

      link.removeAttribute('aria-current');
```

Reply:
It is needed as React can re-use DOM elements when rendering. If we do not remove aria-current from other items, we risk multiple elements have aria-current in the same Breadcrumb component :)